meson_options.txt

```diff
@@ -6,6 +6,8 @@ option('enable-examples', type : 'feature', value : 'enabled', yield : true, des
 option('enable-docs', type : 'feature', value : 'enabled', yield : true, description : 'Build documentation with Doxygen')
 
 # Enable support for different backends
+option('enable-edgetpu', type : 'boolean', value: false,
+       description : 'Enable EdgeTPU backend support')
```

Comment:
Include in the description that needs to have enabled the TFlite backend.
Other way is enable TFlite automatically when you enable the edgetpu.